### PR TITLE
feat: Setup npm publishing

### PR DESCRIPTION
**Pull request by svanherk**
Any steps I missed?

### CODE REVIEW

**Comment by svanherk** [comment]
FYI @dlockhart... I don't know why but GitHub refuses to let me add you as a reviewer 😕

File: README.md [hunk 1]
```diff
@@ -32,17 +40,14 @@ Collection of (search) facet, filter, and sort components
...
-<script src="node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
-<script type="module" src="node_modules/@brightspace-ui-labs/d2l-facet-filter-sort/components/filter-dropdown/filter-dropdown.js"></script>
-<script type="module" src="node_modules/@brightspace-ui-labs/d2l-facet-filter-sort/components/filter-dropdown/filter-dropdown-category.js"></script>
-<script type="module" src="node_modules/@brightspace-ui-labs/d2l-facet-filter-sort/components/filter-dropdown/filter-dropdown-option.js"></script>
+<script type="module">
+	import '@brightspace-ui-labs/facet-filter-sort/components/filter-dropdown/filter-dropdown.js';
+	import '@brightspace-ui-labs/facet-filter-sort/components/filter-dropdown/filter-dropdown-category.js';
+	import '@brightspace-ui-labs/facet-filter-sort/components/filter-dropdown/filter-dropdown-option.js';
+</script>
 ```
```
Comment (dlockhart) [inline README.md:51]: Down under Versioning & Releasing, we typically have [an extra bullet](https://github.com/BrightspaceUI/core#releases):
> Deploy a new package to NPM

File: package.json [hunk 2]
```diff
@@ -5,6 +5,12 @@
     "url": "https://github.com/BrightspaceUILabs/facet-filter-sort.git"
   },
   "name": "@brightspace-ui-labs/facet-filter-sort",
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "/components"
```
Comment (dlockhart) [inline package.json:12]: Include `custom-elements.json` as well so the docs can pick that up.

**Review by dlockhart — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (svanherk) [reply README.md:51]: Right forgot about that!

File: package.json (hunk 2 above)
Comment (svanherk) [reply package.json:12]: Should I delete the file and have it auto-generate before publish like `core` does?

Comment (dlockhart) [reply package.json:12]: Oooh yes that'd be much better!

**Review by dlockhart — APPROVED** [review]

**Pull request merged**